src/main/java/core/basesyntax/Quadrangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public abstract class Quadrangle extends Figure implements Area {
```

Comment:
Don't use abstract classes to set behavior for classes. Consider using an interface if you need to define a contract for classes without providing a default implementation.